### PR TITLE
Remove storage flags and experimental subcommand tests

### PR DESCRIPTION
**Pull request by takirala**
Addresses [DCOS-15193](https://jira.mesosphere.com/browse/DCOS-15193) and partially [DCOS-18769](https://jira.mesosphere.com/browse/DCOS-18769)

### CODE REVIEW

File: cli/dcoscli/data/schemas/manifest-schema.json [hunk 1]
```diff
@@ -1,12 +0,0 @@
-{
```
Comment (takirala) [inline cli/dcoscli/data/schemas/manifest-schema.json]: Is this file used implicitly? I was not able to find how/where this file is used?
@larioj

**Comment by kensipe** [comment]
run linux integration tests

File: cli/setup.py [hunk 2]
```diff
@@ -80,8 +80,7 @@
     package_data={
         'dcoscli': [
             'data/*.json',
-            'data/help/*.txt',
-            'data/schemas/*.json'
```
Comment (kensipe) [inline cli/setup.py]: I think we need to have the /schemas/*.json . why was that removed?

Comment (takirala) [reply cli/setup.py]: @kensipe there were three files inside the `schemas/` and two of them were used [here](https://github.com/dcos/dcos-cli/blob/a6b757fb12b8796a1b292d45cf11bf472e7e4833/cli/dcoscli/experimental/main.py#L162) and [here](https://github.com/dcos/dcos-cli/blob/a6b757fb12b8796a1b292d45cf11bf472e7e4833/cli/dcoscli/experimental/main.py#L186) in `dcos-cli/cli/dcoscli/experimental/main.py` Since I am removing the `dcos-cli/cli/dcoscli/experimental` with this PR i removed those files. I was not able to find any usages for `schemas/manifest-schema.json` 

Thoughts?

^ @larioj after you have added these files there were no modifications. Should we retain these? If so, can you help me understand how they are used?

**Comment by takirala** [comment]
run linux integration tests

File: cli/setup.py (hunk 2 above)
Comment (larioj) [reply cli/setup.py]: These schemas are used for validating the input and output of the build and migrate commands. Unless it is being used elsewhere, they should not be retained.

File: cli/dcoscli/data/schemas/manifest-schema.json (hunk 1 above)
Comment (larioj) [reply cli/dcoscli/data/schemas/manifest-schema.json]: This file may be used for the validation of the manifest created. It fine to be removed.

**Comment by takirala** [comment]
run linux integration tests

**Comment by takirala** [comment]
@jsancio @larioj can you review this? Thanks!

**Comment by takirala** [comment]
@jsancio created https://jira.mesosphere.com/browse/DCOS_OSS-1761

@kensipe Can you review this? Thanks! (linux integration tests were passed in this latest build - https://jenkins.mesosphere.com/service/jenkins/job/DCOS-CLI/job/mesosphere-dcos-cli/job/linux-integration-tests/311/ not sure why github doesn't pick it up)

**Comment by bamarni** [comment]
run all integration tests

**Comment by bamarni** [comment]
Thank you @takirala , always nice to drop some code :)
